pkg/wwan/decryptpasswd/decryptpasswd.go

```diff
@@ -0,0 +1,76 @@
+package main
```

Comment:
Please add the copyright and license lines here.

Reply:
Done. Also the commits were squashed to avoid openssl even being in the git history.

Reply:
Thanks